### ISSUE TITLE
Add ability to pass --force param to box publish command

### ISSUE THREAD

**Issue by bpamiri** [pixl8/github-action-box-publish#1]
I’m using this action in a CI workflow and doing snapshots of the same package version. Unfortunately since the version isn’t being updated the zip file doesn’t get uploaded to Forgebox. Is there anyway to force the update by passing the —force or force=true parameter to the publish command.

This is the results I am getting now:
Contacting ForgeBox...
User [cfwheels] authenticated successfully with [forgebox]
A zip for this version has already been uploaded.  If you want to override the uploaded zip, run this command with the `force` flag.  We will continue to update your package metadata.
Sending package information to forgebox, please wait...
Package is alive, you can visit it here: https://www.forgebox.io/view/cfwheels-base-template
Package published successfully in [forgebox]

**Comment by DominicWatson** [comment]
Hey @bpamiri - any reason you closed this? Sounds like a good question! It's not possible right now, but defo should be.

**Comment by DominicWatson** [comment]
@bpamiri - I've added a force param to the action. You'll need to upgrade to v4 to use it. I've tested that the action still runs successfully with both parameters set (and with none). But would be good to confirm if this works for you where you actually need it. Closing anyway as I think this should be good to go.

Readme is updated with example.

**Comment by bpamiri** [comment]
Dominic,

Thanks for jumping on this so quickly. I opened the issue then thought well I should probably try to fix this and issue a PR back to you, but I didn’t have enough time to work on it over the weekend so I figured I’ll close the ticket figure out how to extend your action and submit the code back. 

Anyway thanks for taking care of this. I did test it and it works beautifully.

-Peter Amiri
 Sent from my iPhone
 (auto-typing mispell disclaimer in effect)
